setup.py

```diff
@@ -21,6 +21,7 @@
         "jinja2",
     ],
     "delayed": [],  # keeping for backwards compatibility
+    "pyodide": ["jinja2"],
```

Comment:
Where is this being used?

Reply:
This is in anticipation of being able to install this in a pyodide session and have the HTML reprs for dataframes, arrays, etc still work. It needs to be separate from the `"diagnostics"` extras since installing `tornado` is still too heavy of a lift.

It's not *hugely* consequential right now, but I have designs to also include [`ipycytoscape`](https://github.com/cytoscape/ipycytoscape) here to replace graphviz in this context (also maybe other contexts, cf. #7301)